sphinx/builders/html/__init__.py

```diff
@@ -865,6 +865,21 @@ def onerror(filename: str, error: Exception) -> None:
                     force=True,
                 )
 
+    def copy_extension_static_files(self) -> None:
+        """Copy static files registered by extensions."""
+        for static_dir in self._registry.static_dirs:
+            if static_dir.is_dir():
+                shutil.copytree(
+                    static_dir,
+                    self._static_dir,
+                    dirs_exist_ok=True,
+                )
+            else:
+                logger.warning(
+                    __('extension static directory %r does not exist'),
```

Comment:
We should print paths as their str representation. 
```suggestion
                    __("extension static directory '%s' does not exist"),
```